front-end/src/components/Sidebar/Sidebar.js

```diff
@@ -1,10 +1,55 @@
-import React from 'react';
-import './style.scss';
+import './Sidebar.css';
+import React, { useState } from 'react';
+import dashboard from '../img/Dashbord.png';
+import users from '../img/3User.png';
+import posts from '../img/Posts.png';
+import categories from '../img/Categories.png';
```

Comment:
@mm375 -san
```JS
import posts from '../img/Posts.png';
```
To avoid misunderstanding with component name you can import as below

```JS
import PostIcon from '../img/Posts.png';
OR use Prefix
import IconPost from '../img/Posts.png';
```
Thanks